src/app/page.tsx

```diff
@@ -1,6 +1,15 @@
 import Image from "next/image";
+import { useEffect, useState } from "react";
 
 export default function Home() {
```

Comment:
anthropic review: The `useState` and `useEffect` hooks are used in a component that appears to be a Server Component, which is not allowed in Next.js 13+ App Router.